eclipse-projects/tdml/.classpath

```diff
@@ -1,96 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-        <!-- This file is updated by the UpdateEclipseClasspath app. -->
-        <classpathentry kind="src" path="src/main/scala"/><classpathentry kind="src" path="src/main/scala-debug"/><classpathentry kind="con" path="org.scala-ide.sdt.launching.SCALA_CONTAINER"/><classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/><classpathentry combineaccessrules="false" kind="src" path="/daffodil-core"/><classpathentry combineaccessrules="false" kind="src" path="/daffodil-lib"/><classpathentry combineaccessrules="false" kind="src" path="/daffodil-runtime1"/><classpathentry combineaccessrules="false" kind="src" path="/daffodil-io"/><classpathentry combineaccessrules="false" kind="src" path="/daffodil-macro-lib"/><classpathentry kind="output" path="target/eclipse/classes"/><!--
```

Comment:
Was this just missed in the other patchset? I feel like this keep changing all the time? Maybe we need a different solution? Maybe they shouldn't be tracked by git and we just clearly document how to create these files if you want to use eclipse?

Reply:
No this is just perfecting these things over time. E.g., as we get rid of these scala-new dirs and such all the eclipse projects have to be updated. Also they were setup wrong at one point and we only discovered this by finding out that searching was generating many hits that are really all the same file but from different modules, etc. In theory this will settle down again and then these won't change again for a while.